ando/tools/generator/utils.py

```diff
@@ -0,0 +1,86 @@
+import copy
+from pathlib import Path
+import pandas as pd
+import json
+
+
+def save_tsv(dataframe, path_to_save):
+    """
+    Append or create a tsv file corresponding of dataframe data
+    Parameters
+    ----------
+    dataframe: dataframe
+        dataframe to save in a TSV format
+    path_to_save: str
+        path to save the TSV file
+
+    """
+    if Path(path_to_save).exists():
+        df = pd.read_csv(path_to_save, sep='\t')
+        output = df.append(dataframe, sort=True)
+        output.to_csv(path_to_save, sep="\t", index=False)
+
+    else:
+        dataframe.to_csv(path_to_save, sep="\t", index=False)
+
+
+def save_json(data_dict, path_to_save):
+    """
+    Append or create a json file corresponding of dict data
+    Parameters
+    ----------
+    data_dict: dict
+        dict to save in a json format
+
+    path_to_save: str
+        path to save the JSON file
+
+    """
+    if Path(path_to_save).exists():
+        with open(path_to_save) as json_file:
+            data_existing = json.load(json_file)
+            new_dict = merge_dict(data_existing, data_dict)
+            json_file.write(new_dict)
```

Comment:
does this works as expected? Why do you use once json for writing and another time the plain file handler? I would expect different layouts for these two mechanisms...

Reply:
Did you check the content of the file?

Reply:
I' ve changed it, that's what made the test fail earlier